doc/releases.md

```diff
@@ -328,3 +328,19 @@ Close your release proposal PR and remove the proposal branch.
 ### 17. Celebrate
 
 _In whatever form you do this..._
+
+
+### 18. ARM6
+
+The ARM6 builds take significantly longer to build than other platforms
+and are more prone to failure. For this reason a release may be published
+in advance of the ARM6 build being ready.  
```

Comment:
This seems like it should be part of the step above:

>If you didn't wait for ARM builds in the previous step before promoting the release, you should re-run `tools/release.sh` after the ARM builds have finished. That will move the ARM artifacts into the correct location. You will be prompted to re-sign SHASUMS256.txt.

Reply:
I feel it at least should not be after `17. Celebrate` (which implies everything is done). As @gibfahn points out, it's already kind of mentioned in `12. Promote and Sign the Release Builds` but maybe it's not prominent enough because we have frequently forgotten to promote them (e.g. https://github.com/nodejs/nodejs.org/issues/1382).

Also is there any particular reason to include the `6` and not just say `ARM`?

Reply:
It is only the ARMv6 builds that are really slow.